### PR TITLE
Kubectl command renaming (run-container to run and resize to scale)

### PR DESCRIPTION
**Pull request by andronat**
For #8319 

I am not sure about this line: https://github.com/GoogleCloudPlatform/kubernetes/blob/master/pkg/kubectl/cmd/util/factory.go#L96

I also took the liberty to rename RunContainer in kubelet, if it is not needed I can discard it.

/cc @smarterclayton, @ghodss

### CODE REVIEW

**Comment by yujuhong** [comment]
I think keeping the term "container" in some names is more clear to me. It makes it easier to tell whether the function is for a pod or for a container. Others may disagree though. /cc @vmarmol @yifan-gu

**Comment by yifan-gu** [comment]
> I think keeping the term "container" in some names is more clear to me. It makes it easier to tell whether the function is for a pod or for a container. Others may disagree though. /cc @vmarmol @yifan-gu

+1

File: docs/api-conventions.md [hunk 1]
```diff
@@ -104,11 +104,11 @@ Objects that contain both spec and status should not contain additional top-leve
...
-* **conditions**: Conditions represent orthogonal observations of an object's current state. Objects may report multiple conditions, and new types of conditions may be added in the future. Condition status values may be `True`, `False`, or `Unknown`. Unlike the phase, conditions are not expected to be monotonic -- their values may change back and forth. A typical condition type is `Ready`, which indicates the object was believed to be fully operational at the time it was last probed. Conditions may carry additional information, such as the last probe time or last transition time. 
+* **conditions**: Conditions represent orthogonal observations of an object's current state. Objects may report multiple conditions, and new types of conditions may be added in the future. Condition status values may be `True`, `False`, or `Unknown`. Unlike the phase, conditions are not expected to be monotonic -- their values may change back and forth. A typical condition type is `Ready`, which indicates the object was believed to be fully operational at the time it was last probed. Conditions may carry additional information, such as the last probe time or last transition time.
```
Comment (bgrant0607) [inline docs/api-conventions.md]: Did anything significant change in this file? If not, please revert it.

**Comment by bgrant0607** [comment]
Please don't change kubelet. #8319 is only about kubectl.

File: docs/api-conventions.md (hunk 1 above)
Comment (andronat) [inline docs/api-conventions.md]: Just empty spaces in the end of the line. should I discard them?

**Comment by andronat** [comment]
Dropping kubelet changes.

**Comment by dchen1107** [comment]
Please update your pr title with more meaningful information. Thanks!

File: docs/api-conventions.md (hunk 1 above)
Comment (bgrant0607) [inline docs/api-conventions.md]: Yes, please don't make insubstantial changes.

**Comment by andronat** [comment]
Put trailing spaces back and rebased to resolve conflicts

File: docs/proposals/autoscaling.md [hunk 2]
```diff
@@ -95,109 +95,109 @@ Since an auto-scaler is a durable object it is best represented as a resource.
...
-        MinAutoScaleCount int 
-                       
-        //TargetSelector provides the resizeable target(s).  Right now this is a ReplicationController
-        //in the future it could be a job or any resource that implements resize.  
+
+        //MinAutoScaleCount defines the minimum number replicas that the auto scaler can reduce to,
+        //0 means that the application is allowed to idle
+        MinAutoScaleCount int
+
+        //TargetSelector provides the scaleable target(s).  Right now this is a ReplicationController
```
Comment (bgrant0607) [inline docs/proposals/autoscaling.md]: scalable

**Comment by bgrant0607** [comment]
Great, thanks. LGTM. We need to wait for the e2e test to be fixed before we can merge, unfortunately.

**Comment by bgrant0607** [comment]
ok to test

**Comment by andronat** [comment]
any news on that? It is ready to merge, I rebased it to latest master

**Comment by thockin** [comment]
needs rebase

**Comment by andronat** [comment]
done

**Pull request merged**